setup.py

```diff
@@ -10,7 +10,7 @@ def text_from_file(path):
     "coverage",
     "isort",
     "jsonschema",
-    "pytest",
+    "pytest<8.0",
```

Comment:
Consider adding a comment to specify why we are restricting pytest -- it's the only dep with a version restriction and the reason is probably not obvious.

```suggestion
    "pytest<8.0", # pytest >= 8 is incompatible with pytest-black
```